nextcord/user.py

```diff
@@ -318,8 +339,10 @@ def __init__(self, *, state: ConnectionState, data: UserPayload) -> None:
 
     def __repr__(self) -> str:
         return (
-            f"<ClientUser id={self.id} name={self.name!r} discriminator={self.discriminator!r}"
-            f" bot={self.bot} verified={self.verified} mfa_enabled={self.mfa_enabled}>"
+            f"<ClientUser id={self.id} name={self.name!r} global_name={self.global_name!r}"
+            f" discriminator={self.discriminator!r}"
+            if self.discriminator != "0"
+            else "" f" bot={self.bot} verified={self.verified}" f" mfa_enabled={self.mfa_enabled}>"
```

Comment:
```suggestion
            f"<ClientUser id={self.id} name={self.name!r} global_name={self.global_name!r}"
            + (f" discriminator={self.discriminator!r}" if self.discriminator != "0" else "") 
            + f" bot={self.bot} verified={self.verified}" f" mfa_enabled={self.mfa_enabled}>"
```

Reply:
If the suggestion is to change it to `+`, why is the last break kept?
`}" f" mfa`

Reply:
My bad, I didn't see that extra break there.